pkg/resource/unstructured/composed/composed.go

```diff
@@ -89,6 +89,20 @@ func (cr *Unstructured) SetConditions(c ...xpv1.Condition) {
 	_ = fieldpath.Pave(cr.Object).SetValue("status.conditions", conditioned.Conditions)
 }
 
+// GetStatus of this Composed resource.
+func (cr *Unstructured) GetStatus() xpv1.Status {
+	status := xpv1.Statused{}
+	if err := fieldpath.Pave(cr.Object).GetValueInto("status", &status); err != nil {
```

Comment:
I am concerned about `status.conditions` and `.status` being stored independently.